test/Test/Toml/Gen.hs

```diff
@@ -285,8 +285,8 @@ genUniHex8Color = do
     pure . Text.pack $ "\\U" ++ hex
 
 -- | Generates text from different symbols.
-genText :: MonadGen m => m Text
-genText =  fmap Text.concat $ Gen.list (Range.constant 0 256) $ Gen.choice
+genText :: (MonadGen m, GenBase m ~ Identity) => m Text
+genText = Gen.filter invalidEscape $ fmap Text.concat $ Gen.list (Range.constant 0 256) $ Gen.choice
```

Comment:
Issue #198 is about not generating `\\` inside TOML keys. However, I believe that `genText` is used not only for keys but for something else. So filtering here is not exactly correct. Instead, you should filter only when generating TOML keys.

Reply:
I think `genText` is not used to generate keys but I also thought that a generic text should not end with a single escape like `"aaa\"`